src/Type/Constant/ConstantStringType.php

```diff
@@ -200,6 +200,9 @@ public function getCallableParametersAcceptors(ClassMemberAccessAnswerer $scope)
 				if (!$scope->canCallMethod($method)) {
 					return [new InaccessibleMethod($method)];
 				}
+				if (!$method->isStatic()) {
+					return [new InaccessibleMethod($method)];
+				}
```

Comment:
I wanted to get this added into `canCallMethod`, which would be in `OutOfClassScope`, but again the context of the class being static or not is unknown and had other side effects.